staging/src/k8s.io/client-go/kubernetes/typed/core/v1/node_expansion.go

```diff
@@ -41,3 +45,16 @@ func (c *nodes) PatchStatus(nodeName string, data []byte) (*v1.Node, error) {
 		Into(result)
 	return result, err
 }
+
+// ProxyGet returns a response of the node by calling it through the proxy.
+func (c *nodes) ProxyGet(scheme, name, port, path string, params map[string]string) rest.ResponseWrapper {
+	request := c.client.Get().
+		Prefix("proxy").
+		Resource("nodes").
+		Name(net.JoinSchemeNamePort(scheme, name, port)).
```

Comment:
It supports it. https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/node/strategy.go#L177